Backpack/BottomSheet/Classes/BottomSheet.swift

```diff
@@ -179,7 +181,9 @@ public final class BPKBottomSheet: NSObject {
             assertionFailure("present(_:animated:completion:) not compatible with persistent presentation style")
             return
         }
-        if let scrollView = floatingPanelController.scrollView {
+        
+        // ToODO --> WHAT IS THIS?
+        if let scrollView = floatingPanelController.trackingScrollView {
```

Comment:
why do we have this?
can we just remove it and leave it as """""native"""""? aka: leave the scrollview offset as is

Reply:
Oh no, we need this for our custom bottom view. I need to remove the todo. woops